client-device-connection-infinispan/src/main/java/org/eclipse/hono/deviceconnection/infinispan/client/HotrodBasedDeviceConnectionInfo.java

```diff
@@ -201,9 +206,12 @@ public HotrodBasedDeviceConnectionInfo(final RemoteCache<String, String> cache,
     public Future<JsonObject> getCommandHandlingAdapterInstances(final String tenantId, final String deviceId,
             final Set<String> viaGateways, final SpanContext context) {
 
+        final Span span = createSpan("get command handling instances", tenantId, deviceId, null, context);
```

Comment:
get command handling *adapter* instances?

otherwise, it will be hard to guess what this span is for, won't it?

Reply:
I've changed it accordingly.